### PR TITLE
How to get/recover chunked elements

### PR DESCRIPTION
**Pull request by Paul-Cornell**
Parity in the API docs with some similar content over in open source.

See https://unstructured-53-docs-42-chunking.mintlify.app/api-reference/how-to/get-chunked-elements

### CODE REVIEW

File: api-reference/how-to/get-chunked-elements.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: Get chunked elements
+---
+
+## Task
+
+You want to get, and print or save, the contents of elements that have been chunked.
+
+You want to get, and print or save, the elements that were originally used to derive a chunked element.
```
Comment (MKhalusova) [inline api-reference/how-to/get-chunked-elements.mdx:9]: It appears that the two sentences are saying the same thing, no?
We should probably leave only one of them.

File: snippets/concepts/chunking-stategy-by-title.mdx [hunk 2]
```diff
@@ -19,5 +10,4 @@ In addition to the behaviors of the `basic` strategy above, the `by_title` strat
...
 
-*   **Combine small sections.** In certain documents, partitioning may identify a list-item or other short paragraph as a `Title` element even though it does not serve as a section heading. This can produce chunks substantially smaller than desired. This behavior can be mitigated using the `combine_text_under_n_chars` argument. This defaults to the same value as `max_characters` such that sequential small sections are combined to maximally fill the chunking window. Setting this to `0` will disable section combining.
-
+*   **Combine small sections.** In certain documents, partitioning may identify a list-item or other short paragraph as a `Title` element even though it does not serve as a section heading. This can produce chunks substantially smaller than desired. This behavior can be mitigated using the `combine_text_under_n_chars` argument. This defaults to the same value as `max_characters` such that sequential small sections are combined to maximally fill the chunking window. Setting this to `0` will disable section combining.
```
Comment (MKhalusova) [inline snippets/concepts/chunking-stategy-by-title.mdx:13]: I didn't know that it defaults to the `max_characters` value! That's a valuable insight.

**Review by MKhalusova — APPROVED** [review]
Overall looks great, there's one minor comment to address, and then feel free to merge.

File: api-reference/how-to/get-chunked-elements.mdx (hunk 1 above)
Comment (Paul-Cornell) [inline api-reference/how-to/get-chunked-elements.mdx:9]:
```suggestion
You want to get, and print or save, the contents of elements that have been chunked.
```

**Pull request merged**